src/common/util.js

```diff
@@ -38,6 +38,10 @@ function getPenultimate(arr) {
   return null;
 }
 
+/**
+ * @param {string | RegExp} chars
+ * @returns {(text: string, index: number, opts?: object) => any}
```

Comment:
Would it make sense to try to type `opts` more exactly than `object`?

Reply:
Yes, definitive 
Also `node` could be typed more exactly

But I would/want to do this 1. later and 2. in another PR

Reply:
Why in a different PR?

Reply:
1. later => It is `01:47 AM` at my location 😅
2. want to improve the types step by step
  and so others (like @brodybits) can work on this and help :)

Reply:
I would really favor a TBD comment in a case like this.

Unfortunately I cannot promise to work on anything specific within the near future due to some other priorities.

I would be happy to answer any questions you may have, though.